<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tilinmuokkausDialog</class>
 <widget class="QDialog" name="tilinmuokkausDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tilin muokkaus</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="tiliRadio">
         <property name="text">
          <string>Tili</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="otsikkoRadio">
         <property name="text">
          <string>Otsikko</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="varoitusKuva">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../pic/pic.qrc">:/pic/varoitus.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="varoitusLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tilinumeron muuttaminen aiheuttaa muutoksia jo kirjatuissa vienneissä. Jos tili on valittu vastatiliksi, katoaa tämä valinta. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Numero</string>
       </property>
       <property name="buddy">
        <cstring>numeroEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="numeroEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nimi</string>
       </property>
       <property name="buddy">
        <cstring>nimiEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nimiEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="tasoLabel">
       <property name="text">
        <string>Otsikkotaso</string>
       </property>
       <property name="buddy">
        <cstring>tasoSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="tasoSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tilityyppi</string>
       </property>
       <property name="buddy">
        <cstring>tyyppiCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="tyyppiCombo"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Vastatili</string>
       </property>
       <property name="buddy">
        <cstring>vastatiliEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="TilinvalintaLine" name="vastatiliEdit"/>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="veroCombo"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="verolajiLabel">
       <property name="text">
        <string>Verolaji</string>
       </property>
       <property name="buddy">
        <cstring>veroCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="veroprosenttiLabel">
       <property name="text">
        <string>Veroprosentti</string>
       </property>
       <property name="buddy">
        <cstring>veroSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="veroSpin">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="poistoaikaLabel">
       <property name="text">
        <string>Poistoaika</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="poistoaikaSpin">
       <property name="suffix">
        <string>vuotta</string>
       </property>
       <property name="maximum">
        <number>1200</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="poistoprossaLabel">
       <property name="text">
        <string>Menojäännöspoisto</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="poistoprossaSpin">
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Täydentävä teksti</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="taydentavaEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="teGroup">
     <property name="title">
      <string>Tase-erittely</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="taseEratRadio">
        <property name="text">
         <string>Täysi erittely tase-erillä</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="teLiVaRadio">
        <property name="text">
         <string>Lisäykset ja vähennykset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="teSaldoRadio">
        <property name="text">
         <string>Saldo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="teEiRadio">
        <property name="text">
         <string>Ei näy tase-erittelyssä</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Kirjausohje</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="kirjausohjeText"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TilinvalintaLine</class>
   <extends>QLineEdit</extends>
   <header>db/tilinvalintaline.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tilinmuokkausDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tilinmuokkausDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otsikkoRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>tasoLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otsikkoRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>tasoSpin</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tiliRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>tyyppiCombo</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tiliRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tiliRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tiliRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>vastatiliEdit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
